diff --git a/build/gulpfile.reh.ts b/build/gulpfile.reh.ts
index a40ff6f6..7d0139d5 100644
--- a/build/gulpfile.reh.ts
+++ b/build/gulpfile.reh.ts
@@ -51,6 +51,7 @@ const BUILD_TARGETS = [
 	{ platform: 'darwin', arch: 'arm64' },
 	{ platform: 'linux', arch: 'x64' },
 	{ platform: 'linux', arch: 'arm64' },
+        { platform: 'linux', arch: 'arm' },
 	{ platform: 'alpine', arch: 'arm64' },
 	// legacy: we use to ship only one alpine so it was put in the arch, but now we ship
 	// multiple alpine images and moved to a better model (alpine as the platform)
@@ -250,6 +251,9 @@ function fetchNodejsFromInternalFeed(feed: string, assetName: string, version: s
 
 BUILD_TARGETS.forEach(({ platform, arch }) => {
 	task.task(task.define(`node-${platform}-${arch}`, () => {
+            if (platform === 'linux' && arch === 'arm') {
+                    return Promise.resolve(null);
+            }
 		const nodePath = path.join('.build', 'node', `v${nodeVersion}`, `${platform}-${arch}`);
 
 		if (!fs.existsSync(nodePath)) {
